e2e/tests/transfer/localhost_test.go

```diff
@@ -50,6 +50,18 @@ func (s *LocalhostTransferTestSuite) TestMsgTransfer_Localhost() {
 
 	s.Require().NoError(test.WaitForBlocks(ctx, 1, chainA), "failed to wait for blocks")
 
+	t.Run("verify begin blocker was executed", func(t *testing.T) {
+		cs, err := s.QueryClientState(ctx, chainA, exported.LocalhostClientID)
+		s.Require().NoError(err)
+		originalHeight := cs.GetLatestHeight()
+
+		s.Require().NoError(test.WaitForBlocks(ctx, 1, chainA), "failed to wait for blocks")
+
+		cs, err = s.QueryClientState(ctx, chainA, exported.LocalhostClientID)
+		s.Require().NoError(err)
+		s.Require().True(cs.GetLatestHeight().GT(originalHeight), "client state height was not incremented")
```

Comment:
we could also check that latest height == ctx height

Reply:
do you mean `ctx`? We we don't have access to the height the way we do in the unit tests, let me know if I'm misunderstanding the suggestion!

Reply:
This is true, I guess you'd have to query for the node status and pull the latest height from there. I'm totally happy with the current approach, feel free to resolve

Reply:
I think at the moment this test will always fail if begin blocker is not called, so I think it is fine as is!